src/main/scala/ch/epfl/bluebrain/nexus/admin/projects/Projects.scala

```diff
@@ -319,8 +320,16 @@ object Projects {
 
   private[projects] object Eval {
 
+    private def endsWithHashOrSlash(iri: AbsoluteIri): Boolean =
+      iri.asString.matches(".*(#|/)$")
+
+    private def invalidIriMessage(param: String) =
+      s"the Iri on the '$param' parameter should end with hash (#) or slash (/)/"
```

Comment:
replace the message with:
"the value of the project's '$param' parameter must end with hash (#) or slash (/)"